examples/raffle/index.rsh

```diff
@@ -152,4 +152,4 @@ export const main =
         }))
       .timeout(deadline, () => closeTo(Sponsor, () => {}));
       commit();
-    });
+    });
```

Comment:
Nitpick: can you set your editor to keep a newline at the end of the file?

Reply:
i got you